pizzakit-plugin/includes/pizzakit.php

```diff
@@ -135,8 +135,8 @@ private static function create_payment($order){
 
 		if($order_total < 1){
 			global $wpdb;
-			$table = $wpdb->prefix . 'payment';
-			$data = array('orderID' => $order_id,'uuid' => '-2','status'=>'INVALID_TOTAL');
+			$table = $wpdb->prefix . 'orders';
+			$data = array('id' => $order_id,'uuid' => '-2','status'=>'INVALID_TOTAL');
 			$format = array('%d','%s','%s');
 			$wpdb->insert($table,$data,$format);
```

Comment:
This is trying to insert a new row into orders instead of modifying it, causing an error since the id is supposed to be unique. try changing wpdb->insert into something like this https://developer.wordpress.org/reference/classes/wpdb/update/